### DISCUSSION TITLE
ccf-validation-tools/Palatine_Tonsil/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#274] [General]
# ccf-validation-tools/Palatine_Tonsil/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Palatine_Tonsil/comments/

**Comment by emquardokus** [comment]
palatine-tonsil-v1.0-DRAFT
https://docs.google.com/spreadsheets/d/14VTmaZyxa68uEl9sDKE7N8VBGL8893WlrxnTPBhcjBM/edit#gid=0

**Different labels**
  FIXED:   In row 17, the term CL:0000624 has different name/label in the source ontology. The name/label in the ASCT+B table is CD4-positive alpha-beta T cells and the one in the ontology is CD4-positive, alpha-beta T cell. For reference, the given name/label by SMEs is CD4 T cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED:  In row 26, the term UBERON:0001610 has different name/label in the source ontology. The name/label in the ASCT+B table is dorsal lingual artery and the one in the ontology is lingual artery. For reference, the given name/label by SMEs is lingual artery. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED: In row 18, the term CL:0000625 has different name/label in the source ontology. The name/label in the ASCT+B table is CD8-positive alpha-beta T cells and the one in the ontology is CD8-positive, alpha-beta T cell. For reference, the given name/label by SMEs is CD8 T cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Blank ontology ID missing parent**

ADDED CT/1 plasma cell:    In row 25, the term plasma cell CD21 without ontology ID has no parent that is from the CL ontology.

ADDED CT/1 B cell:     In row 36, the term Cycling B without ontology ID has no parent that is from the CL ontology.

ADDED CT/1 B cell:        In row 42, the term Cycling B without ontology ID has no parent that is from the CL ontology.

ADDED germinal center B cell	CL:0000844:     In row 49, the term germinal center B cell without ontology ID has no parent that is from the CL ontology.

ADD CT/1 germinal center B cell	CL:0000844:     In row 50, the term germinal center B cell CD21 without ontology ID has no parent that is from the CL ontology.

ADDED CT/1 B cell:        In row 54, the term Centroblast without ontology ID has no parent that is from the CL ontology.

ADDED CT/1 B cell:        In row 55, the term Cycling B without ontology ID has no parent that is from the CL ontology.

ADDED germinal center B cell	CL:0000844:    In row 62, the term germinal center B cell without ontology ID has no parent that is from the CL ontology.

ADDED CT/1 germinal center B cell    In row 63, the term germinal center B cell CD21 without ontology ID has no parent that is from the CL ontology.

**Comment by anitacaron** [comment]
New term in the ontology:
Palatine tonsil : preplasmablast (CL:4033065)
